packages/sil-voting-app/src/routes/dashboard/components/Proposal/Description.tsx

```diff
@@ -0,0 +1,51 @@
+import Collapse from '@material-ui/core/Collapse';
+import Block from 'medulas-react-components/lib/components/Block';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React, { useState } from 'react';
+import { elipsify } from '../../../../utils/strings';
+
+const DESC_MAX_LENGTH = 180;
+
+interface Props {
+  readonly description: string;
+}
+
+const Description = (props: Props): JSX.Element => {
+  const [expanded, setExpanded] = useState(false);
+
+  const onClick = (): void => {
+    setExpanded(!expanded);
```

Comment:
This is not correct, you can have race conditions here. You should update expanded based on the previous state:
```
setExpanded(prevExpanded => !prevExpanded)
```